Cargo.toml

```diff
@@ -34,6 +34,8 @@ rustls-pemfile = "1"
 hyper = { version = "0.14", features = ["full"] }
 phf = { version = "0.11.1", features = ["macros"] }
 exitcode = "1.1.2"
+postgres-protocol = "0.6.4"
```

Comment:
This is a great find!

Reply:
@magec I created this library a while ago with intention to create a PR to use this in PgCat. Wondering if you'd take a look, https://github.com/zain-kabani/postgres-proto-rs

Reply:
I took this one because was the one used in postgres [crate](https://github.com/sfackler/rust-postgres). Also, I found it pretty easy to use and is widely imused. Havent seen yours yet. Will take a look.